modules/capi-creating-cluster-resource.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cluster_api_machine_management/cluster-api-getting-started.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="capi-creating-cluster-resource_{context}"]
+= Creating the Cluster API cluster resource
+
+You can create the cluster resource by creating a YAML manifest file and applying it with the {oc-first}.
+
+.Prerequisites
+
+* You have deployed an {product-title} cluster.
+
+* You have enabled the use of the Cluster API.
+
+* You have access to the cluster using an account with `cluster-admin` permissions.
+
+* You have installed the {oc-first}.
+
+* You have obtained the cluster ID value.
+
+.Procedure
+
+. Create a `<cluster_resource_file>.yaml` file similar to the following:
++
+--
+[source,yaml]
+----
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  name: <cluster_name> # <1>
+  namespace: openshift-cluster-api
+spec:
+  infrastructureRef:
+    apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+    kind: <infrastructure_kind> # <2>
+    name: <cluster_name>
+    namespace: openshift-cluster-api
+----
+<1> Specify the cluster ID as the name of the cluster.
+<2> Specify the infrastructure kind for the cluster.
+The following values are valid:
++
+* `AWSCluster`: The cluster is running on {aws-first}.
+* `GCPCluster`: The cluster is running on {gcp-first}.
+--
+
+. Create the cluster CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f <cluster_resource_file>.yaml
+----
+
+.Verification
+
+* Confirm that the cluster CR exists by running the following command:
++
+[source,terminal]
+----
+$ oc get cluster
+----
++
+.Example output
+[source,text]
+----
+NAME             PHASE          AGE    VERSION
+<cluster_name>   Provisioning   4h6m
+----
++
+The cluster resource is ready when the value of `PHASE` is `Running`.
```

Comment:
In the testing, the value of PHASE is `Provisioned` when the resource is ready, not sure it's correct or a bug.
```
$ oc get cluster                                                                                                                                           
NAME                CLUSTERCLASS   PHASE         AGE   VERSION
zhsun-aws29-8ksnk                  Provisioned   37m
```